ReText/window.py

```diff
@@ -823,6 +826,17 @@ def openFile(self):
 	def openFileWrapper(self, fileName):
 		if not fileName:
 			return
+		if not QFile.exists(fileName):
+			if self.createFileQuestion(fileName):
+				try:
+					print('Create file' + ' ' + fileName , file=sys.stdout)
+					open(fileName, 'x').close()
+				except:
```

Comment:
Please do not use bare `except:`, always specify exceptions you want to catch.

See https://docs.quantifiedcode.com/python-anti-patterns/correctness/no_exception_type_specified.html